common/cmake/devices.cmake

```diff
@@ -980,31 +985,62 @@ function(ADD_FPGA_TARGET)
   endforeach()
 
   if(NOT ${ADD_FPGA_TARGET_NO_SYNTHESIS})
-    set(
-      COMPLETE_YOSYS_SCRIPT
-      "tcl ${YOSYS_SCRIPT}"
-    )
+    set(COMPLETE_YOSYS_SYNTH_SCRIPT "tcl ${YOSYS_SYNTH_SCRIPT}")
+
+    set(OUT_JSON_SYNTH ${OUT_LOCAL}/${TOP}_synth.json)
+    set(OUT_JSON_SYNTH_REL ${OUT_LOCAL_REL}/${TOP}_synth.json)
+    set(OUT_JSON ${OUT_LOCAL}/${TOP}.json)
+    set(OUT_JSON_REL ${OUT_LOCAL_REL}/${TOP}.json)
 
     add_custom_command(
-      OUTPUT ${OUT_EBLIF} ${OUT_SYNTH_V}
+      OUTPUT ${OUT_JSON_SYNTH} ${OUT_SYNTH_V}
       DEPENDS ${SOURCE_FILES} ${SOURCE_FILES_DEPS}
               ${YOSYS} ${YOSYS_TARGET} ${QUIET_CMD} ${QUIET_CMD_TARGET}
-              ${YOSYS_SCRIPT}
+              ${YOSYS_SYNTH_SCRIPT}
       COMMAND
         ${CMAKE_COMMAND} -E make_directory ${OUT_LOCAL}
       COMMAND
         ${CMAKE_COMMAND} -E env
           symbiflow-arch-defs_SOURCE_DIR=${symbiflow-arch-defs_SOURCE_DIR}
-          OUT_EBLIF=${OUT_EBLIF}
+          OUT_JSON=${OUT_JSON_SYNTH}
           OUT_SYNTH_V=${OUT_SYNTH_V}
           ${ADD_FPGA_TARGET_DEFINES}
-          ${QUIET_CMD} ${YOSYS} -p "${COMPLETE_YOSYS_SCRIPT}" -l ${OUT_EBLIF}.log ${SOURCE_FILES}
+          ${QUIET_CMD} ${YOSYS} -p "${COMPLETE_YOSYS_SYNTH_SCRIPT}" -l ${OUT_JSON_SYNTH}.log ${SOURCE_FILES}
       WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
       VERBATIM
     )
+
+    set(SPLIT_INOUTS ${symbiflow-arch-defs_SOURCE_DIR}/utils/split_inouts.py)
+
+    add_custom_command(
+      OUTPUT ${OUT_JSON}
+      DEPENDS ${OUT_JSON_SYNTH} ${QUIET_CMD} ${QUIET_CMD_TARGET} ${SPLIT_INOUTS} ${PYTHON3} ${PYTHON3_TARGET} simplejson
+      COMMAND
+        ${CMAKE_COMMAND} -E env PYTHONPATH=${PRJXRAY_DIR}:${symbiflow-arch-defs_SOURCE_DIR}/utils ${PYTHON3} ${SPLIT_INOUTS} -i ${OUT_JSON_SYNTH} -o ${OUT_JSON}
```

Comment:
What is PRJXRAY_DIR required here?   It seems weird to have a 7-series specific thing in the (in theory) generic CMake function?

Reply:
Upon review of split_inouts, it doesn't seem to use PRJXRAY at all? So you should just be able to drop the path?

Reply:
Right, It ended up here as a mistake. Removed.